src/cloud-api-adaptor/entrypoint.sh

```diff
@@ -30,6 +30,7 @@ optionals+=""
 [[ "${SECURE_COMMS_PP_OUTBOUNDS}" ]] && optionals+="-secure-comms-pp-outbounds ${SECURE_COMMS_PP_OUTBOUNDS} "
 [[ "${SECURE_COMMS_KBS_ADDR}" ]] && optionals+="-secure-comms-kbs ${SECURE_COMMS_KBS_ADDR} "
 [[ "${PEERPODS_LIMIT_PER_NODE}" ]] && optionals+="-peerpods-limit-per-node ${PEERPODS_LIMIT_PER_NODE} "
+[[ "${DISK_SIZE}" ]] && optionals+="-disk-size ${DISK_SIZE} "
```

Comment:
For AWS provider we use a param "ROOT_VOLUME_SIZE" as a way to expand the VM root volume size if requested explicitly. May be the same can be reused? Or we can settle on another name for all the providers?

Reply:
ROOT_VOLUME_SIZE would be good IMO

Reply:
should it be made a generic param or copied to azure?

Reply:
Afaik it will not be straight forward to implement disk resize for all the providers. So I think copying to azure should be ok for now.